src/v2/view-builder/views/email/ChallengeAuthenticatorDataEmailView.js

```diff
@@ -0,0 +1,39 @@
+import { loc } from 'okta';
+import BaseAuthenticatorEmailView from './BaseAuthenticatorEmailView';
+
+const BaseAuthenticatorEmailForm = BaseAuthenticatorEmailView.prototype.Body;
+
+const Body = BaseAuthenticatorEmailForm.extend(
+  {
+    title() {
+      return loc('oie.email.mfa.title', 'login');
+    },
+
+    save() {
+      return loc('oie.email.verify.primaryButton', 'login');
+    },
+
+    initialize() {
+      BaseAuthenticatorEmailForm.prototype.initialize.apply(this, arguments);
+      const userEmail = this.options.appState.get('currentAuthenticatorEnrollment')?.profile?.email;
+      const subtitleText = loc('oie.email.verify.subtitle', 'login');
+
+      this.add(`<div class="okta-form-subtitle" data-se="o-form-explain">${subtitleText}
+        <span class="strong no-translate">${userEmail}</span>
+      </div>`, {
+        prepend: true,
+        selector: '.o-form-error-container',
```

Comment:
Since this is not an `error` instead use.. `.o-form-info-container`.. 

This `selector: '.o-form-error-container'` might have been used in other pages, but it was kind of hack when `info-container` didn't existed

Reply:
I did try doing that but when `.o-form-info-container` is used as selector the content is not added to the screen. I get blank section like this

![Screen Shot 2021-11-10 at 2 57 08 PM](https://user-images.githubusercontent.com/91922055/141207283-9ade6aee-f63e-42d4-9c85-d194afb3b69e.png)

Reply:
adding the section in `postRender` did the trick